tests/ga/test_cgroupconfigurator.py

```diff
@@ -542,39 +542,37 @@ def test_start_extension_command_should_disable_cgroups_and_invoke_the_command_d
     @patch('time.sleep', side_effect=lambda _: mock_sleep())
     def test_start_extension_command_should_capture_only_the_last_subprocess_output(self, _):
         with self._get_cgroup_configurator() as configurator:
-            pass  # release the mocks used to create the test CGroupConfigurator so that they do not conflict the mock Popen below
+            original_popen = subprocess.Popen
```

Comment:
Recent changes in the cgroup code added new calls to systemctl to set quotas; those calls are already handled by the mocks created in self._get_cgroup_configurator, but previously those mocks were not needed and we were releasing them. Now we use them.